src/shared/messages.js

```diff
@@ -11,7 +11,7 @@ import { EventEmitter } from 'events';
  * @returns {{log: AriaATCIShared.Log<T>, emitter: EventEmitter}}
  * @template T
  */
-export function createLogger(messages) {
+export function _createLogger(messages) {
```

Comment:
Renamed this to `createSharedLogger`. I was updating names of some things to avoid `import {thing as prefixedThing}` cases. I chose `_createLogger` I think to in part indicate that its use is restricted to `aria-at-harness`, and rename it for when `src/agent/messages.js` and `src/host/messages.js` import it without having to rename it.